test/e2e/runner/load.go

```diff
@@ -27,73 +25,101 @@ func Load(ctx context.Context, testnet *e2e.Testnet, multiplier int) error {
 	if concurrency == 0 {
 		concurrency = 1
 	}
-	initialTimeout := 1 * time.Minute
-	stallTimeout := 30 * time.Second
 
 	chTx := make(chan types.Tx)
-	chSuccess := make(chan types.Tx)
+	chSuccess := make(chan int)
```

Comment:
```suggestion
	chSuccess := make(chan int)  // success counts per iteration
```
(just so it's clear what the int means)